src/sentry/conf/server.py

```diff
@@ -1071,8 +1071,6 @@ def create_partitioned_queues(name):
     "organizations:sso-scim": False,
     # Enable workaround for migrating IdP instances
     "organizations:sso-migration": False,
-    # Enable transaction comparison view for performance.
-    "organizations:transaction-comparison": False,
```

Comment:
Removing in this PR since the frontend one will have removed all the checks already